libs/portal/features/profile/ui/questions.stories.tsx

```
@@ -0,0 +1,29 @@
+      ]);
+
+      return remixStub({ initialEntries: ['/'] });
+    },
+  ],
+};
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+// Default View of the Component
+export const Default: Story = {};
```

ESLint

[error] Extra semicolon.

(style/semi)